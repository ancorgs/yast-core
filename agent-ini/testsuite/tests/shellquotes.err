[Interpreter] tests/shellquotes.ycp:XXX Unquoting error: missing closing quote. Unquoted value: <"invalid - unclosed>.
