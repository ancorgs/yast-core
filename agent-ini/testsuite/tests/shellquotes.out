([])
Val1=string
Val2=string1string2
Val3='shell_quoted_value'\''s "speciality"'
Val5='string with space'
Val6='string with escaped spaces'
Val7='funny quotes'
Val4=''\''written'\'' "value"'
